mix.exs

```diff
@@ -101,6 +101,8 @@ defmodule Astarte.Flow.MixProject do
   # Add here additional documentation files
   defp docs do
     [
+      main: "0001-overview",
+      extra_section: "GUIDES",
```

Comment:
I'm not really sure about the purpose of `extra_section`, however I don't think you need uppercase, ex_doc does it for you.

Reply:
This basically makes the extra pages fall under "Guides" instead of "Pages". I copied the capitalization from Phoenix and Ecto but it works even with a normal capitalization, fixing